### ISSUE TITLE
appmake +zx colliding with binaries of previous builds

### ISSUE THREAD

**Issue by zx70** [z88dk/z88dk#2539]
a_BSS.bin   
a_DATA.bin  

Error: Section  overlaps section DATA by 7317 bytes
Error: Section DATA overlaps section BSS by 677 bytes
zx: Aborting... one or more binaries overlap
Building application code failed

**Comment by zx70** [comment]
@pauloscustodio I wonder if this could be addressed by Appmake ?
Is it too risky to begin by wiping <progname>_*.bin on the output folder ?

**Comment by pauloscustodio** [comment]
> Is it too risky to begin by wiping _*.bin on the output folder ?

Yes, I think so. We don't know what the user might have in the folder.

What is the test case that triggers that error?

**Comment by zx70** [comment]
If you build a program needing sections it will create more than a single file (a.bin, a_bss.bin, etc..).
If a different program is later built appmake tries to gather also the previous files, I thought it was using the .map, but apparently things go differently. 
I got this behaviour with zcc +zx -create-app..
